frontend/src/index.html

```diff
@@ -1,249 +1,354 @@
 <!DOCTYPE html>
 <html lang="en" class="text-gray-600 font-sans bg-gray-50">
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <meta name="theme-color" content="#115E59" />
-    <meta name="msapplication-navbutton-color" content="#115E59 " />
-    <link rel="manifest" href="./manifest.webmanifest" />
-    <link rel="shortcut icon" href="./favicon.png" />
-    <title>Pure Bhakti Archive</title>
-  </head>
 
-  <body class="relative flex flex-col overlayed:overflow-hidden sm:h-screen">
-    <header class="bg-header pb-0.5 w-full bg-clip-content border-b-4 border-dotted border-peach-dark">
-      <div class="relative mx-auto max-w-screen-xl">
-        <div class="bg-header-decor absolute left-0 top-2 w-11 mobile-m:w-12 mobile-l:w-14 h-full bg-contain bg-left-top bg-no-repeat mobile-s:opacity-100 opacity-25 md:top-0 md:w-24 md:h-20"></div>
-        <div
-          class="bg-header-decor absolute right-0 top-2 w-11 mobile-m:w-12 mobile-l:w-14 h-full bg-contain bg-right-top bg-no-repeat mobile-s:opacity-100 opacity-25 transform -scale-x-1 md:top-0 md:w-24 md:h-20"
-        ></div>
-        <div
-          class="relative z-10 px-1 py-3 text-center text-teal-800 font-header mobile-l:text-base mobile-m:text-sm text-xs font-semibold tracking-tighter sm:text-lg md:tracking-normal lg:ml-60 lg:mr-28"
-        >
-          <span class="sm:flex sm:justify-center sm:space-x-3">
-            <span class="bg-lotus hidden flex-none w-7 bg-contain bg-right-top bg-no-repeat sm:block"></span>
-            <span>
-              <span class="mobile-s:block mobile-s:whitespace-nowrap sm:inline">The Official Archive of</span>
-              <span class="mobile-s:whitespace-nowrap">His Divine Grace</span>
-            </span>
-            <span class="bg-lotus hidden flex-none w-7 bg-contain bg-right-top bg-no-repeat transform -scale-x-1 sm:block"></span>
-          </span>
-          <span class="lg:text-xl">
-            <span class="whitespace-nowrap">Sri Srimad</span>
-            <span class="mobile-s:block mobile-s:whitespace-nowrap sm:inline">Bhaktivedanta Narayana Gosvami Maharaja</span>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <meta name="theme-color" content="#115E59" />
+  <meta name="msapplication-navbutton-color" content="#115E59 " />
+  <link rel="manifest" href="./manifest.webmanifest" />
+  <link rel="shortcut icon" href="./favicon.png" />
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
We don't want to use font icons. Inline SVG should be used, as everywhere in the code. Look around.

Reply:
Sure i will use svg elements

Reply:
And please add comments above the icons to specify which particular icon is used. As you can see in this HTML file for the other icons.